### PR TITLE
Portfolio development

### PR DESCRIPTION
**Pull request by CalebBaines1**
Personalize portfolio in 3 commits by:

- Updating index.html with personal details

- Adding picture of myself

- Changing the addRandomGreeting function to addRandomFact

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
   <body>
     <div id="content">
       <h1>Caleb Baines Portfolio</h1>
+      <img src="./images/caleb_pic.jpg" alt="Picture of Caleb"
+      width="200" height="225"
+      />
```
Comment (blakewilliams1) [inline portfolio/src/main/webapp/index.html:14]: Feel free to condense these 3 lines down to one, it looks like there's enough space

**Comment by blakewilliams1** [comment]
Lgtm. I also tried to suggest an indenting edit but but feel free to take it or leave it.

**Review by blakewilliams1 — APPROVED** [review]
lgtm

**Review by CalebBaines1 — COMMENTED** [review]
Thanks, I agree.

Blue moon is definitely Midwestern and one of the few flavors I feel does not need toppings.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (ruixia) [reply portfolio/src/main/webapp/index.html:14]: +1

**Review by ruixia — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (CalebBaines1) [reply portfolio/src/main/webapp/index.html:14]:
```suggestion
     <img src="./images/caleb_pic.jpg" alt="Picture of Caleb" width="200" height="225" />
```

**Review by CalebBaines1 — COMMENTED** [review]
Just barely 2 lines but I think that works.

**Pull request merged**